sdk/helper/ldaputil/config.go

```diff
@@ -559,3 +561,51 @@ func (c *ConfigEntry) Validate() error {
 	}
 	return nil
 }
+
+func ConvertConfig(cfg *ConfigEntry) *capldap.ClientConfig {
+	// cap/ldap doesn't have a notion of connection_timeout, and uses a single timeout value for
+	// both the net.Dialer and ldap connection timeout.
+	// So take the smaller of the two values and use that as the timeout value.
+	minTimeout := min(cfg.ConnectionTimeout, cfg.RequestTimeout)
+	urls := strings.Split(cfg.Url, ",")
+	config := &capldap.ClientConfig{
+		URLs:                                 urls,
+		UserDN:                               cfg.UserDN,
+		AnonymousGroupSearch:                 cfg.AnonymousGroupSearch,
+		GroupDN:                              cfg.GroupDN,
+		GroupFilter:                          cfg.GroupFilter,
+		GroupAttr:                            cfg.GroupAttr,
+		UPNDomain:                            cfg.UPNDomain,
+		UserFilter:                           cfg.UserFilter,
+		UserAttr:                             cfg.UserAttr,
+		ClientTLSCert:                        cfg.ClientTLSCert,
+		ClientTLSKey:                         cfg.ClientTLSKey,
+		InsecureTLS:                          cfg.InsecureTLS,
+		StartTLS:                             cfg.StartTLS,
+		BindDN:                               cfg.BindDN,
+		BindPassword:                         cfg.BindPassword,
+		AllowEmptyPasswordBinds:              true,
```

Comment:
Wondering if this should be `cfg.DenyNullBind` instead of true?

Reply:
yeah I think that's correct. or more accurately, `!cfg.DenyNullBind`. Updated!